python/spider-py/src/spider_py/storage/mariadb_storage.py

```diff
@@ -356,6 +348,34 @@ def create_driver(self, driver_id: core.DriverId) -> None:
             self._conn.rollback()
             raise StorageError(str(e)) from e
 
+    def _create_data_with_ref(self, data: core.Data, insert_ref: str, ref_id: UUID) -> None:
+        """
+        Inserts data object into the database with reference.
+        :param data: The data object to insert.
+        :param insert_ref: The SQL statement to insert the reference.
+        :param ref_id: The reference ID (DriverId or TaskId).
+        :raises StorageError: If the insertion fails.
+        """
```

Comment:
```suggestion
    def _create_data_with_ref(self, data: core.Data, insert_stmt: str, ref_id: UUID) -> None:
        """
        Creates a data object in the storage with the given `ref_id` references to the data.
        :param data: The data object to create.
        :param insert_stmt: The SQL statement to insert the reference.
        :param ref_id: The reference ID (DriverId or TaskId).
        :raises StorageError: If the storage operations fail.
        """
```
- `insert_ref` is a bad name: it looks like a reference but it's not. How about `insert_stmt`?
- We should make the docstring consistent: I rewrote it in the similar way as `create_data_with_xxx_ref` in `Storage`.